config/settings.py

```
@@ -5,17 +5,48 @@
 
 """
 
-from os import environ as enviornment_var
 import os
+import site
+import sys
+from os import environ as environment_var
 from pathlib import Path
+from typing import List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)